cmd/logsJob.go

```diff
@@ -122,10 +124,12 @@ func getLogsJob(cmd *cobra.Command, apiClient *apiclient.Radixapi, appName, jobN
 	}
 }
 
-func getSteps(apiClient *apiclient.Radixapi, appName, jobName string) []*models.StepLog {
+func getSteps(apiClient *apiclient.Radixapi, appName, jobName string, sinceTime time.Time) []*models.StepLog {
+	since, _ := strfmt.ParseDateTime(sinceTime.Format(time.RFC3339))
 	jobLogParameters := job.NewGetApplicationJobLogsParams()
 	jobLogParameters.SetAppName(appName)
 	jobLogParameters.SetJobName(jobName)
+	jobLogParameters.SetSinceTime(&since)
```

Comment:
Will this pointer not point to the stack as "sinceTime" is passed by value? I hope SetSinceTime does not keep the reference after returning

Reply:
jobLogParameters requires that it is passed as pointer. That is not something I can do something with. Then again since is a local variable to the function, constructed on every call, so the code is not affected by any possible mutation

Reply:
I was not fearing mutation but a pointer to a value that will be removed once the function is returned, but I hope go has built-in safeguards for this.